app/.nvmrc

```diff
@@ -1 +1 @@
-16.20.2
\ No newline at end of file
+18.17.0
```

Comment:
Forge cli was complaining I was using an old version and was having issues running forge commands. Had to bump, which came parcelled with a need to change this.